packages/qunit-dom/package.json

```diff
@@ -50,7 +45,19 @@
     "testPathIgnorePatterns": [
       "/node_modules/",
       "/dist/"
-    ]
+    ],
+   "extensionsToTreatAsEsm": [".ts"],
+    "moduleNameMapper": {
+      "^(\\.{1,2}/.*)\\.js$": "$1"
+    },
+    "transform": {
+      "^.+\\.ts?$": [
+        "ts-jest",
```

Comment:
`jest` was replaced by `vitest`.
Is this still needed?

Reply:
this PR (#2065) does not build on top of: https://github.com/mainmatter/qunit-dom/pull/2066
To reduce rebase headache, I'm going to rebase #2065 (this one) after #2066 is accepted/merged